docs/api-reference/next.config.js/headers.md

```diff
@@ -4,6 +4,8 @@ description: Add custom HTTP headers to your Next.js app.
 
 # Headers
 
+> This feature was introduced in [Next.js 9.5](https://nextjs.org/blog/next-9-5) and up. If you’re using older versions of Next.js, please upgrade before trying it out.
```

Comment:
Should we mention this for `basePath` and `trailingSlash` also?

Reply:
Yes! 🐛